runtime-testsuite/test/org/antlr/v4/test/runtime/go/BaseTest.java

```diff
@@ -464,13 +516,17 @@ private String locateGo() {
 		return prop;
 	}
 
-	private String locateRuntime() {
+	private static File locateRuntime() {
 		final ClassLoader loader = Thread.currentThread().getContextClassLoader();
 		final URL runtimeSrc = loader.getResource("Go");
```

Comment:
I am afraid, I can't give an educated answer. All I know is Peter needed the path to the Go runtime source code and did whatever it takes to make it available. Or possibly the ANTLR guy provided that lookup? I didn't question it, just used it. :)